lms/djangoapps/edxnotes/views.py

```diff
@@ -66,6 +68,32 @@ def edxnotes(request, course_id):
     return render_to_response("edxnotes/edxnotes.html", context)
 
 
+@require_GET
+@login_required
+def list_notes(request, course_id):
```

Comment:
Do we really need two separate views? If the structure of the response is the same, can we just do with one view?